binlog_streamer.go

```diff
@@ -151,42 +160,60 @@ func (s *BinlogStreamer) Run() {
 			continue
 		}
 
+		evPosition := mysql.Position{
+			Name: currentFilename,
+			Pos:  ev.Header.LogPos,
+		}
+
+		s.logger.WithFields(logrus.Fields{
+			"position":                   evPosition.Pos,
+			"file":                       evPosition.Name,
+			"type":                       fmt.Sprintf("%T", ev.Event),
+			"lastStreamedBinlogPosition": s.lastStreamedBinlogPosition,
+		}).Debug("reached position")
```

Comment:
This is going to be very verbose, no?

Reply:
Also, the "first" event ghostferry receive will have `file` here as `""`, correct?

Reply:
It's at `Debug` level. Our default logging level is `Info` AFAICT: https://github.com/Shopify/ghostferry/blob/eab1fec72d0081fabe8663676e702569e4671faa/vendor/github.com/sirupsen/logrus/logger.go#L26-L29

Reply:
Here's an excerpt of the logs in the beginning:

```
D, [2020-04-16T13:19:11.875657 #84026] DEBUG -- : stderr: {"level":"info","msg":"starting data iterator run","tablesCount":1,"tag":"data_iterator","time":"2020-04-16T13:19:11-04:00"}
D, [2020-04-16T13:19:11.875922 #84026] DEBUG -- : stderr: {"level":"info","msg":"starting binlog streamer","tag":"binlog_streamer","time":"2020-04-16T13:19:11-04:00"}
D, [2020-04-16T13:19:11.875971 #84026] DEBUG -- : stderr: {"level":"info","msg":"starting periodic reverifier","tag":"inline-verifier","time":"2020-04-16T13:19:11-04:00"}
D, [2020-04-16T13:19:11.877298 #84026] DEBUG -- : stderr: {"file":"mysql-bin.004297","lastStreamedBinlogPosition":{"Name":"mysql-bin.004297","Pos":816},"level":"debug","msg":"reached position","position":0,"tag":"binlog_streamer","time":"2020-04-16T13:19:11-04:00","type":"*replication.RotateEvent"}
D, [2020-04-16T13:19:11.877467 #84026] DEBUG -- : stderr: {"filename":"mysql-bin.004297","level":"info","msg":"binlog file rotated, advancing lastStreamedBinlogPosition","pos":816,"tag":"binlog_streamer","time":"2020-04-16T13:19:11-04:00"}
D, [2020-04-16T13:19:11.877627 #84026] DEBUG -- : stderr: {"file":"mysql-bin.004297","lastStreamedBinlogPosition":{"Name":"mysql-bin.004297","Pos":816},"level":"debug","msg":"reached position","position":0,"tag":"binlog_streamer","time":"2020-04-16T13:19:11-04:00","type":"*replication.FormatDescriptionEvent"}
D, [2020-04-16T13:19:11.877772 #84026] DEBUG -- : stderr: {"file":"mysql-bin.004297","lastStreamedBinlogPosition":{"Name":"mysql-bin.004297","Pos":816},"level":"debug","msg":"reached position","position":881,"tag":"binlog_streamer","time":"2020-04-16T13:19:11-04:00","type":"*replication.GTIDEvent"}
D, [2020-04-16T13:19:11.883181 #84026] DEBUG -- : stderr: {"file":"mysql-bin.004297","lastStreamedBinlogPosition":{"Name":"mysql-bin.004297","Pos":881},"level":"debug","msg":"reached position","position":949,"tag":"binlog_streamer","time":"2020-04-16T13:19:11-04:00","type":"*replication.QueryEvent"}
```

the first `file` would be set correctly, as we start the binlog streamer with the master position set in `Init()`